pipelines/build/common/build_base_file.groovy

```diff
@@ -181,7 +181,7 @@ class Builder implements Serializable {
             ENABLE_TESTDYNAMICPARALLEL: enableTestDynamicParallel,
             ENABLE_INSTALLERS: enableInstallers,
             ENABLE_SIGNER: enableSigner,
-            CLEAN_WORKSPACE: cleanWorkspace,
+            CLEAN_WORKSPACE_BEFORE: cleanWorkspaceBefore,
```

Comment:
cleanWorkspaceBefore vs cleanWsAfter for the variable name, please pick one for consistency (I prefer the full name :-))

Reply:
@karianna fixed, can you please review again ?